src/03-modality-agnostic-files.md

```diff
@@ -60,9 +59,37 @@ field_info = {
 }
 
 table = render.make_metadata_table(schema_obj, field_info)
+```
+
+try markdown
+
+```{code-cell} ipython3
 display(Markdown(table.to_markdown()))
```

Comment:
Another option if we add `markdown` as a dependency:

```suggestion
from markdown import markdown

display(HTML(markdown(table.to_markdown())))
```

Reply:
I just tried that out locally. It looks the same as the first Markdown test. 😞

Reply:
What about 

```suggestion
from markdown import markdown

display(HTML(markdown(table.to_markdown(), extensions=['tables']))))
```

Reply:
:tada: it looks like it works!!!!

Reply:
Take a look: https://bids-standard.github.io/bids-specification-ignore/src/03-modality-agnostic-files.html#dataset-description-json

Reply:
wait a second... it looks like it's missing a bunch of rows. but it looks good in my local build. 😕

Reply:
okay, I was looking at the wrong table.

Reply:
It's because I forgot to add `markdown` to the requirements 🤦